e2e/tests/components/date-range-picker/date-range-picker.e2e-spec.ts

```diff
@@ -298,6 +298,23 @@ describe('Date Range Picker Tests', () => {
         expect(await imageCompare('date-range-format')).toEqual(0);
     });
 
+    it('should update date picker when input date is updated', async () => {
+
+        // Clear input and enter new date
+        await page.input.clear();
+        await page.input.sendKeys('1 January 2020  09:00 AM GMT — 3 January 2020  10:59 PM GMT');
+        expect(await page.getInputText()).toBe('1 January 2020  09:00 AM GMT — 3 January 2020  10:59 PM GMT');
+
+        // remove focus from input
+        await page.clickOnRangeHeader();
+
+        // Check date picker has been updated
+        expect(await page.getPickerDateHeader(Picker.Start)).toBe('1 January 2020');
+        expect(await page.getPickerTimeHeader(Picker.Start)).toBe('9:00 AM');
+        expect(await page.getPickerDateHeader(Picker.End)).toBe('3 January 2020');
+        expect(await page.getPickerTimeHeader(Picker.End)).toBe('10:59 PM');
```

Comment:
Can we check that the time spinners (hours, minutes, seconds) also update when the input is changed, since there was a fix for that in this PR? The above is mainly testing the parsing code in the test page component, which doesn't really give us any coverage.